README.md

```diff
@@ -56,8 +56,14 @@ In addition, there are some Yarn scripts defined in the root package as a conven
 
 `yarn dev:pd` — starts the development servers for the Spotlight API and the Public Dashboard Client in a single terminal window. Be sure you have configured each of those applications with the necessary environment variables, as described in their README files, or this will not work!
 
+`yarn dev:spotlight` - similar to the above, but with Spotlight Client as the frontend.
+
 `yarn demo:pd` — starts the development servers for the Spotlight API and the Public Dashboard client in "demo mode" by supplying the necessary environment variables in the command line. (This will run your local site off of fake data fixtures rather than live data from a calculation pipeline; see the [Spotlight API documentation](spotlight-api/#demo-mode) for more information.)
 
+`yarn demo:spotlight` - again, similar to the above, with with Spotlight Client as the frontend.
+
+`yarn test` - runs the tests for all packages in a single command. (The output this produces is pretty messy at the moment but it can be useful for editor integration)
```

Comment:
You may want to add a note here that the demo:spotlight server needs to be running to run any tests.

Reply:
it shouldn't! There is some global test setup code in `spotlight-client/globalTestSetup.js` that actually starts a test server for you (and corresponding code in `spotlight-client/globalTeardown.js` to shut it down when testing is finished). But if you find that isn't actually working for you locally then .... we have a problem. (it seems to work in CI though so hopefully that would just be resolved by getting one's environment set up properly)

Reply:
When I run `yarn test:sc` without the server runnign I get `FetchError: request to http://localhost:3001/api/US_ND/public failed, reason: connect ECONNREFUSED 127.0.0.1:3001`

Reply:
even with the url set to port 3002 in `env.test`?

Reply:
That worked now this time around right away without any change. 🤷‍♀️